clang-tools-extra/unittests/clang-doc/HTMLGeneratorTest.cpp

```diff
@@ -30,7 +30,7 @@ ClangDocContext
 getClangDocContext(std::vector<std::string> UserStylesheets = {},
                    StringRef RepositoryUrl = "") {
   ClangDocContext CDCtx{
-      {}, "test-project", {}, {}, {}, RepositoryUrl, UserStylesheets};
+      {}, "test-project", {}, {}, {}, {}, {}, RepositoryUrl, UserStylesheets};
```

Comment:
If the new parameters go at the end, and have default values, then you don't need to updated this.